modules/message-lane/src/lib.rs

```diff
@@ -136,6 +134,66 @@ pub trait Config<I = DefaultInstance>: frame_system::Config {
 	type MessageDispatch: MessageDispatch<Self::InboundMessageFee, DispatchPayload = Self::InboundPayload>;
 }
 
+/// Extended weight info.
+pub trait WeightInfoEx: WeightInfo {
```

Comment:
A bit of a nitpick here, but I'd prefer if the trait was declared after all the pallet macros. I don't think this need to be one of the first things somebody reads when looking at the pallet.

Reply:
I've extracted this trait into dedicated file